pkg/schedule/filter/filters.go

```diff
@@ -332,6 +332,8 @@ type StoreStateFilter struct {
 	// If it checks failed, the operator will be put back to the waiting queue util the limit is available.
 	// But the scheduler should keep the same with the operator level.
 	OperatorLevel constant.PriorityLevel
+	// set true if not allow balance leader from this store
+	ForbidSplit bool
```

Comment:
How about changing to another name? It sounds strange.

Reply:
has fixed as `ForbidSplit RecentlySplitRegions `